src/js/Components/RacingBoards.js

```diff
@@ -0,0 +1,80 @@
+import Events from '../Constants/Events.js'
+import { Congratulation } from '../Constants/Message.js'
+import Component from '../Core/Component.js'
+import { SET_MANUAL } from '../modules/actions.js'
+import { displayWinners, moveCars, setManual } from '../modules/creator.js'
+import { store } from '../modules/store.js'
+import { getRandomInt } from '../utils/random.js'
+
+export default class RacingBoards extends Component {
+  constructor(target) {
+    super(target)
+  }
+
+  setEvent(target) {}
+
+  dispatchMoveCars() {
+    setTimeout(() => {
+      store.dispatch(moveCars())
+    }, 1000)
+  }
+
+  template() {
+    const { cars, totalAttemps, remainAttemps, manual } = store.getState()
+
+    const gameEnded = remainAttemps === 0
+
+    if (totalAttemps <= 0) return ''
+
+    if (!manual) {
+      if (remainAttemps > 0) {
+        this.dispatchMoveCars()
+      }
+    }
+
+    if (gameEnded) {
+      const max = Math.max(...cars.map((car) => car.move))
+      const gameWinners = cars
+        .filter((car) => car.move === max)
+        .map((car) => car.name)
+
+      store.dispatch(displayWinners(gameWinners))
+
+      setTimeout(() => {
+        alert(Congratulation)
+      }, 2000)
+    }
+
+    return `
+    <div class="mt-4 d-flex" id="board">
+        ${cars
+          .map(({ name, move }) => {
+            const $DOWNS = []
+            for (let i = 0; i < move; i++) {
+              $DOWNS.push(`<div class="forward-icon mt-2">⬇️️</div>`)
+            }
```

Comment:
선언형 html template에서 절차형 로직을 작성하신 이유가 무엇인가요?

Reply:
똑같은 `DOM`을 `move` 카운트만큼 로딩하려고 하다 보니 저렇게 구현했는데 혹시 더 좋은 방법이 있을까요? 🤔

Reply:
```suggestion
            {new Array(move).fill(`<div class="forward-icon mt-2">⬇️️</div>`).join('')}
```
과제하다보니 저도 비슷한 상황이 나왔네요ㅎ
이렇게 접근하는것도 한 방법일꺼 같습니다